BUILD.bazel

```diff
@@ -1573,25 +1573,7 @@ cc_library(
 )
 
 # torch
-py_binary(
-    name = "stringify_file",
-    srcs = ["torch/csrc/jit/codegen/cuda/tools/stringify_file.py"],
-)
-
-generated_nvfuser_hdrs = ["generated_" + hdr for hdr in libtorch_nvfuser_generated_headers]
```

Comment:
Sorry the PR is massive, so I may something obvious -- are we dropping nvfuser from the bazel build completely? That doesn't seem right...

Reply:
Yes we are doing that in this PR.

> Since we moved nvfuser into a cmake build, we effectively disabled bazel build for nvfuser. This could impact internal workload at Meta, so we need to put support back. cc'ing @vors

We do plan to put it back though. Bazel build is... not easily tested and verified, so I'm trying to do that in a separate PR, otherwise, this one is going to be dragging forever... :cry:

Reply:
David started a separate PR disabling nvfuser in bazel workload #91836 , just to make sure that it's safe to do so in this PR.

Reply:
@jjsjann123 @vors actually, my PR is only disabling nvfuser in the _internal_ bazel build; not in the OSS bazel build.

Reply:
Thx for clarifying~